stubs/FortifyServiceProvider.php

```diff
@@ -18,7 +18,9 @@ class FortifyServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        if (env('MAIL_USERNAME', null) && env('MAIL_PASSWORD', null)) {
```

Comment:
You can't use the `env` helper here directly as it won't read ENV values when config is cached.